configserver/src/main/java/com/yahoo/vespa/config/server/maintenance/ApplicationPackageMaintainer.java

```diff
@@ -94,7 +94,10 @@ protected double maintain() {
                                     return;
                                 }
                             }
-                            catch (Exception ignored) { }
+                            catch (Exception e) {
+                                log.info("Exception when downloading application package (" + appFileReference + ")" +
```

Comment:
No success is already logged at info level at line 102.
I would just make this a warning right away as that stands out more and is easier to detect.

Reply:
Makes sense, done, need a new review